lib/Doctrine/Persistence/ManagerRegistry.php

```diff
@@ -61,7 +61,7 @@ public function getAliasNamespace(string $alias) : string;
     /**
      * Gets all object manager names.
      *
-     * @return string[] An array of object manager names.
+     * @return array<string, string> An array of object manager names.
```

Comment:
Similar to the above: is this really indexed by name?

Reply:
Yes, I thought it was weird too but I didn't want to change the behavior in this PR.